wandb/sdk/launch/runner/kubernetes.py

```diff
@@ -180,33 +209,34 @@ def run(self, launch_project: LaunchProject) -> Optional[AbstractRun]:  # noqa:
             container_resources["limits"] = resource_args.get("resource_limits")
         if container_resources:
             containers[0]["resources"] = container_resources
-        # todo: args and env vars would be added here, need to figure out what kind of overrides we want
-
-        containers[0]["security_context"] = {
-            "allowPrivilegeEscalation": False,
-            "capabilities": {
-                "drop": ["ALL"]
-            },
-            "seccompProfile": {
-                "type": "RuntimeDefault"
+        for c in containers:
+            c["security_context"] = {
+                "allowPrivilegeEscalation": False,
+                "capabilities": {"drop": ["ALL"]},
+                "seccompProfile": {"type": "RuntimeDefault"},
             }
-        }
 
-        pod_spec["restartPolicy"] = resource_args.get("pod_restart_policy", "Never")
-        if resource_args.get("pod_preemption_policy"):
-            pod_spec["preemptionPolicy"] = resource_args.get("pod_preemption_policy")
-        if resource_args.get("node_name"):
-            pod_spec["nodeName"] = resource_args.get("node_name")
-        if resource_args.get("node_selectors"):
-            pod_spec["nodeSelectors"] = resource_args.get("node_selectors")
+        # env vars
+        given_env_vars = resource_args.get("env", {})
+        env_vars = get_env_vars_dict(launch_project, self._api)
+        merged_env_vars = {**env_vars, **given_env_vars}
+        containers[0]["env"] = [
+            {"name": k, "value": v} for k, v in merged_env_vars.items()
+        ]
 
+        # cmd
+        entry_point = launch_project.get_single_entry_point()
```

Comment:
This will break the no build case, where an entrypoint is not specified

Reply:
ok yeah this is broken on every runner, fixing all